reference/architecture-explained.md

```diff
@@ -35,13 +35,13 @@ There are two key aspects in MACH Architecture: decoupling and composability. De
 ![MACH Alliance Reference architecture diagram medium detail, platform level](<../src/diagrams/MACH Alliance Reference Architecture Diagram - medium.png>)
 
 
-### Integration Layers secure and why
+### Horizontal Integration Layers
```

Comment:
@adampnielsen I reverted back to the title from Google Docs just because the new one didn't seem to belong, but feel free to suggest a better option.